Documentation/Signoffs/Motor System.md

```diff
@@ -0,0 +1,303 @@
+^ There will be 20 dividers per platform, equating to 60 total platforms.  
+
+* **Given these values, each divider will weigh 0.27375 lbs (0.365 * 0.75). The total weight of the dividers will be 16.425 lbs.**  
+
+**The hollow cylinder supporting the platforms:**  
+* **Will be made of aluminum or pvc (assuming aluminum for worst-case)**  
+
+* **Will be 36- to 40-inches tall (assuming 40 inch for worst-case).**  
+
+* **Will be 1/16th or 1/8th thick (assuming 1/8th inch for worst-case).**  
+^ Given the density of aluminum (2.7 g/cm^3) and given the area equation for a hollow cylinder  
+
+$M = (π)(h)(mD)(r^2 - (r - t)^2)$  
+where:  
+  M = mass of the hollow cylinder
+  h = height  
+  r = outer radius  
+  t = thickness  
+  mD = Mean Density of cylinder material  
+The hollow aluminum cylinder will weigh **9.622 lbs**  
+
+**The Motor:**
+* **Given the calculations above, must rotate a total weight of 70.858 lbs. (We will assume roughly 20% tolerance for error, so for calculations we will use 85 lbs)**
+
+* **Will have a time to accelerate (the loads) of less than 1 second.**
+^ As explained below, we want our time to accelerate to be 0.5 seconds. We can run our calculations based off of 1 second time to accelerate as a worse-case scenario.
+
+**Circuity:** 
+* **Will be protected at 125% the nameplate current rating (7.5A).** This comes from the following codes.
+
+The standard NEC 430.52(D) states "Torque motor branch circuits shall be protected at the motor nameplace current rating in accordance with 240.4(B)
+
+^ Source: https://up.codes/s/rating-or-setting-for-individual-motor-circuit
+
+NEC 240.4(B) is the standard for overcurrent devices rated 800 amperes or less, which allows the use of a higher rated overcurrent device as long as the conditions are met. All of these conditions have been cleared.  
+ * The conductors being protected are not part of a branch circuit supplying more than one receptacle for cord-and-plug-connected portable loads.  
+ * The ampacity of the conductors does not correspond with the standard ampere rating of a fuse or a circuit breaker without overload trip adjustments above its rating (but that shall be permitted to have other trip or rating adjustments).  
+ * The next higher standard rating selected does not exceed 800 amperes.  
+^ Source: https://up.codes/viewer/illinois/nfpa-70-2020/chapter/2/wiring-and-protection#240.4_(B)  
+
+The standard NEC 705.12(D)(2) Bus or Conductor Ampere Rating states "One hundred twenty-five percent [125%] of the inverter output circuit current shall be used in ampacity calculations for the following: Feeders, Taps, and Busbars."
+
+^ Source: https://up.codes/viewer/delaware/nfpa-70-2014/chapter/7/special-conditions#705.12_(D)_(2)
+
+## Schematic
+
+![Schematic](https://github.com/DillonSW/Capstone_Team_5/blob/Team5-signoff-Motor-System/images/DriverAndMotorSchemRevision.jpg)
```

Comment:
If the microcontroller is to be selected as a part of another design, then it should not appear here. Rather, it is sufficient to label the wires and to have outbound jumps to the micro/PLC. On the other hand, it seems you have a theoretical encoder. What is going on there? I would expect to see a part number and analysis regarding its accuracy if it is a part of this design. If it is not, how do you know that the motor can position accurately enough?